build.gradle

```diff
@@ -22,7 +22,7 @@ sourceCompatibility = 1.14
 targetCompatibility = 1.14
 
 allprojects {
-    version = '0.11.0'
+    version = '0.11.1'
```

Comment:
Let's do a minor version bump since this contains new features

Reply:
++